action.yml

```diff
@@ -2,6 +2,9 @@ name: "Install a binary from GitHub releases"
 description: "Install binaries from GitHub releases"
 author: "Lee Briggs"
 inputs:
+  token:
+    description: 'GITHUB_TOKEN or a `repo` scoped Personal Access Token (PAT)'
+    default: ${{ github.token }}
```

Comment:
Defaults to `GITHUB_TOKEN` so users don't need to specify a token in workflows unless they want to access a private repository.